Add support for provisioning on a specific cluster (#42)
chef/kitchen-vcenter: README.md

 - `folder` - Folder into which the new machine should be stored. If specified the folder _must_ already exist.
 - `poweron` - Power on the new virtual machine. Default: true
 - `vm_name` - Specify name of virtual machine. Default: `<suite>-<platform>-<random-hexid>`

Only one of the following optional parameters can be given:

>  - `resource_pool` - Name of the resource pool to use when creating the machine. Will use first pool by default.

Reviewer: ```suggestion
 - `resource_pool` - Name of the resource pool to use when creating the machine. Default: first pool
```

>  - `cluster` - Cluster on which the new virtual machine should be created. Will provision in the root pool, if paramter given and choose a targethost within

Reviewer: ```suggestion
 - `cluster` - Cluster on which the new virtual machine should be created. Default: Provisions in the root pool. If a parameter is given,  if parameter given, then chooses a `targethost` within.
```
I'm not sure I understand what `cluster` does, particularly when passed with a parameter.

Author: Yeah, that description was definitively too fuzzy. 

What about "Default: picks first cluster found in vCenter"
